GitUI/MessageBoxes.cs

```diff
@@ -127,6 +130,11 @@ public static bool ConfirmUpdateSubmodules(IWin32Window? owner)
             return result;
         }
 
+        public static bool ConfirmBranchCheckout(IWin32Window? owner)
+        {
+            return Confirm(owner, Instance._confirmBranchCheckout.Text, Instance._confirmBranchCheckoutCaption.Text);
+        }
```

Comment:
```suggestion
        => Confirm(owner, string.Format(Instance._confirmBranchCheckout.Text, branchName), Instance._confirmBranchCheckoutCaption.Text);
```
with `branchName` as additional argument